pages/03_server_management_and_restapi.md

```diff
@@ -0,0 +1,46 @@
+# User Management, Content Management, and Display with the REST API
```

Comment:
Done. But I have no link. Is there a good link to information about CORS support?